src/lib/y2storage/boot_requirements_strategies/legacy.rb

```diff
@@ -148,11 +156,10 @@ def errors_on_gpt
         errors = []
 
         if grub_part_needed_in_gpt? && missing_partition_for?(grub_volume)
-          message = _(
-            "There is no partition of type BIOS Boot. " \
-            "For GPT devices it is strongly advised to use a BIOS Boot partition. " \
-            "A setup without such a partition is not supported and may cause " \
-            "problems now or in the future."
+          # TRANSLATORS: %s is a partition type, e.g. "BIOS Boot"
+          message = format(
+            _("A partition of type %s is needed to install the bootloader."),
+            PartitionId::BIOS_BOOT.to_human_string
```

Comment:
We intentionally introduced this specific message (mentioning problems in the future) recently to address the complaints about enforcing a BIOS Boot partition (by users who managed to make their systems boot without it). Why do we want to revert the message now?